src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -333,4 +336,27 @@ public boolean equals(Object obj) {
                 && filteredBlacklistedPersons.equals(other.filteredBlacklistedPersons);
     }
 
+
+    //==================== Event Handling Code ===============================================================
+
+    @Subscribe
+    public void handleLoginUpdateDebt(LoginAppRequestEvent event) {
```

Comment:
it would be more consistent to name this handleLoginAppRequestEvent, and extract a method for updating the debt

Reply:
is it possible to have two subscribing functions of the same name? My method is also called `handleLoginAppRequestEvent`

Reply:
yes it is, thanks to `polymorphism` ^^